<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="8" passed="7">
  <reporter-output>
  </reporter-output>
  <suite name="RegressionSuite" duration-ms="51752" started-at="2020-07-25T11:42:03Z" finished-at="2020-07-25T11:42:55Z">
    <groups>
    </groups>
    <test name="RegressionTestCases" duration-ms="51752" started-at="2020-07-25T11:42:03Z" finished-at="2020-07-25T11:42:55Z">
      <class name="com.trendyol.tests.MainTest">
        <test-method status="PASS" signature="initialization()[pri:0, instance:com.trendyol.tests.MainTest@16aa0a0a]" name="initialization" is-config="true" duration-ms="11257" started-at="2020-07-25T11:42:03Z" finished-at="2020-07-25T11:42:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialization -->
        <test-method status="PASS" signature="verifyHomePageTitleTest()[pri:1, instance:com.trendyol.tests.MainTest@16aa0a0a]" name="verifyHomePageTitleTest" duration-ms="3287" started-at="2020-07-25T11:42:14Z" finished-at="2020-07-25T11:42:18Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyHomePageTitleTest -->
        <test-method status="FAIL" signature="verifyUserLoginTest()[pri:2, instance:com.trendyol.tests.MainTest@16aa0a0a]" name="verifyUserLoginTest" duration-ms="11292" started-at="2020-07-25T11:42:18Z" finished-at="2020-07-25T11:42:29Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at com.trendyol.tests.MainTest.verifyUserLoginTest(MainTest.java:55)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyUserLoginTest -->
        <test-method status="PASS" signature="searchTest()[pri:3, instance:com.trendyol.tests.MainTest@16aa0a0a]" name="searchTest" duration-ms="162" started-at="2020-07-25T11:42:29Z" finished-at="2020-07-25T11:42:29Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- searchTest -->
        <test-method status="PASS" signature="clickRandomProductTest()[pri:4, instance:com.trendyol.tests.MainTest@16aa0a0a]" name="clickRandomProductTest" duration-ms="9472" started-at="2020-07-25T11:42:29Z" finished-at="2020-07-25T11:42:39Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- clickRandomProductTest -->
        <test-method status="PASS" signature="addToCartProductTest()[pri:5, instance:com.trendyol.tests.MainTest@16aa0a0a]" name="addToCartProductTest" duration-ms="3569" started-at="2020-07-25T11:42:39Z" finished-at="2020-07-25T11:42:42Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addToCartProductTest -->
        <test-method status="PASS" signature="checkProductPriceAndCartPrice()[pri:6, instance:com.trendyol.tests.MainTest@16aa0a0a]" name="checkProductPriceAndCartPrice" duration-ms="3213" started-at="2020-07-25T11:42:42Z" finished-at="2020-07-25T11:42:45Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkProductPriceAndCartPrice -->
        <test-method status="PASS" signature="addToCartNumberTest()[pri:7, instance:com.trendyol.tests.MainTest@16aa0a0a]" name="addToCartNumberTest" duration-ms="4356" started-at="2020-07-25T11:42:45Z" finished-at="2020-07-25T11:42:50Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addToCartNumberTest -->
        <test-method status="PASS" signature="deleteProductInCartTest()[pri:8, instance:com.trendyol.tests.MainTest@16aa0a0a]" name="deleteProductInCartTest" duration-ms="5068" started-at="2020-07-25T11:42:50Z" finished-at="2020-07-25T11:42:55Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- deleteProductInCartTest -->
      </class> <!-- com.trendyol.tests.MainTest -->
    </test> <!-- RegressionTestCases -->
  </suite> <!-- RegressionSuite -->
</testng-results>
